src/nft/components/collection/ActivityCells.tsx

```diff
@@ -203,23 +217,23 @@ export const PriceCell = ({ marketplace, price }: { marketplace?: Markets; price
 }
 
 interface EventCellProps {
-  eventType: ActivityEventType
+  eventType: NftActivityType
   eventTimestamp?: number
   eventTransactionHash?: string
   eventOnly?: boolean
-  price?: string
+  price?: string | number
```

Comment:
wondering if we can cast this to a string in the hook above just to make it easier to deal with. Since these values are somewhat large I find dealing with strings always better